src/search/services/search.service.ts

```diff
@@ -37,6 +39,26 @@ export class SearchService {
     return window.sessionStorage.getItem(key) ? JSON.parse(window.sessionStorage.getItem(key)) : null;
   }
 
+  public decrementStartRecord(): void {
```

Comment:
This should return the new start record - because where ever it is used, the first thing you do is query it

Reply:
@andywilkinshmcts - as above... it had crossed my mind.